src/main/kotlin/com/ermolnik/Application.kt

```diff
@@ -1,22 +1,19 @@
 package com.ermolnik
 
-import com.ermolnik.core.HOST_URL
-import com.ermolnik.core.PORT
-import com.ermolnik.core.plugins.*
-import com.ermolnik.db.configureDatabases
+import com.ermolnik.core.plugins.configureHTTP
+import com.ermolnik.core.plugins.configureMonitoring
+import com.ermolnik.core.plugins.configureRouting
+import com.ermolnik.core.plugins.configureSerialization
+import com.ermolnik.db.configureDatabase
 import io.ktor.server.application.*
-import io.ktor.server.engine.*
-import io.ktor.server.netty.*
 
-fun main() {
-    embeddedServer(Netty, port = PORT, host = HOST_URL, module = Application::module)
-        .start(wait = true)
-}
+fun main(args: Array<String>) = io.ktor.server.netty.EngineMain.main(args)
```

Comment:
а для чего ты поменял? в чем профит от нетти?

Reply:
We're still using Netty here, the same server engine:
- `embeddedServer(Netty...`
+ `io.ktor.server.netty...`

The only change is the way of creating a server in our code, because embeddedServer is too constrained in terms of flexibility and customization and is suitable only for prototypes and rapid development when you pass all the parameters in code.
The main reason for change to `EngineMain` is the ability to use configuration file, we will need to be able to load a configuration file and environment variables for deployment anyways.

See https://ktor.io/docs/engines.html#choose-create-server